extension/kettle/impl/pom.xml

```diff
@@ -110,6 +110,11 @@ under the License.
       <artifactId>log4j</artifactId>
       <version>1.2.17</version>
     </dependency>
+      <dependency>
+          <groupId>org.json</groupId>
+          <artifactId>json</artifactId>
+          <version>20240303</version>
+      </dependency>
```

Comment:
Is it possible to use Jackson directly, so that there is no need to introduce a new pom?